packages/frontend/src/components/VisualizationConfigs/TableConfigPanel/ConditionalFormattingList.tsx

```diff
@@ -120,6 +120,17 @@ const ConditionalFormattingList = ({}) => {
                     variant="contained"
                     value={openItems}
                     onChange={handleAccordionChange}
+                    styles={(theme) => ({
+                        control: {
+                            padding: theme.spacing.xs,
+                        },
+                        label: {
+                            padding: 0,
+                        },
+                        panel: {
+                            padding: 0,
+                        },
+                    })}
```

Comment:
Added these `styles` to match the other accordion in ReferenceLines! Just to make them _the same_